doc/plutus-core-spec/notation.tex

```diff
@@ -17,7 +17,7 @@ \subsection{Sets}
   \item $\B^*$ is the set of all bytestrings.
   \item $\Z = \{\ldots, -2, -1, 0, 1, 2, \ldots\}$.
   \item $\mathbb{F}_q$ denotes a finite field with $q$ elements ($q$ a prime power).
-  \item $\mathbb{F}_q^*$ denotes the multiplicative group of nonzero elements of $\mathbb{F}_q$.
+  \item $\units{\mathbb{F}} = \mathbb{F}\setminus\{0\}$ denotes the multiplicative group of nonzero elements of a field $\mathbb{F}$.
```

Comment:
This is to avoid confusion with the X^* notation for sequences of elements of a set X.  Both F^* and F^× are used in the mathematical literature.